lib/bpk-component-text-link/stories.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-console */
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+
+import CenterDecorator from '../../storybook/CenterDecorator';
+import BpkText from '../bpk-component-text';
+import BpkThemeProvider from '../bpk-theming';
+import themeAttributes from '../../storybook/themeAttributes';
+import { WEIGHT_STYLES } from '../bpk-component-text/src/common-types';
+
+import BpkTextLink from './index';
+
+storiesOf('bpk-component-text-link', module)
+  .addDecorator(CenterDecorator)
+  .add('default', () => (
+    <BpkText>
+      Hello World! This{' '}
+      <BpkTextLink
+        accessibilityLabel="Link to press"
+        onPress={() => console.log('Clicked the link')}
+      >
+        link
+      </BpkTextLink>{' '}
+      is clickable!
+    </BpkText>
+  ))
+  .add('with-mixed-styles', () => (
+    <BpkText textStyle="lg" weight={WEIGHT_STYLES.heavy}>
```

Comment:
This is using the wrong combination of style and weight, `lg` is not supported with `heavy` weight. Let's change this to emphasised. I also think we should change this story to use `lg` and `emphasized` for the link as well, I don't see this case being very common, if the link is in the middle of the text it would normally have the same style. I would change this to `With different text style`

Reply:
Sure thing no problem!